build_system/seqan3-config.cmake

```diff
@@ -151,17 +151,19 @@ endmacro ()
 # ----------------------------------------------------------------------------
 # Detect if we are a clone of repository and if yes auto-add submodules
 # ----------------------------------------------------------------------------
+# Note that seqan3-config.cmake can be standalone and thus SEQAN3_SRC_DIR might be empty.
+find_path(SEQAN3_SRC_DIR NAMES build_system/seqan3-config.cmake HINTS "${CMAKE_CURRENT_LIST_DIR}/..")
```

Comment:
if we do include a shortcut for this, I would prefer "SEQAN3_CLONE_DIR", because that underlines that it is only set for clone, not when seqan is instaled system-wide.